src/styles/fonts.scss

```
@@ -5,6 +5,13 @@
   font-weight: normal;
 }
 
+@font-face {
+  font-family: "Unhinted Rounded M+ 1p";
+  src: url("../fonts/unhinted-rounded-mplus-1p-medium.woff2");
+
+  font-weight: medium;
+}
+
```

ソフトウェア全体のフォントweightがmediumになってて変だなと思ったんですが、どうやら`font-face`の`font-weight`にmediumはないっぽかったです！
https://developer.mozilla.org/ja/docs/Web/CSS/@font-face/font-weight#%E5%80%A4

なので多分こうしないとダメかもです。
これでダイアログ内の本文はレギュラーに、メニューアイテムなどはmediumになりました！
```suggestion
@font-face {
  font-family: "Unhinted Rounded M+ 1p";
  src: url("../fonts/unhinted-rounded-mplus-1p-medium.woff2");

  font-weight: 500;
}

```
スクリーンショットの置き換えが必要なのが違和感あったのですが、これによって全てのフォントweightが変わっちゃってたのが原因な気がしますね！

ちなみに右クリックメニューやキャラクター選択メニューなどもmediumになってます。
この辺意図した挙動かどうかお聞きしたいかもです！

---

@Hiroshiba 
ほんとうでした！ありがとうございます、こちら修正します

---

こちら修正済みです